src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,44 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import { User } from '../../types/User';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+export type PostInfoProps = {
+  post: PostWithComments;
+};
+
+export type PostWithComments = {
```

Comment:
This could be dependent on the Post type without the need for rewriting properties:
<img width="257" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts/assets/87986955/d3a377b2-01b2-410b-bc2d-e1bb7dac58c1">

or with interface
<img width="321" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts/assets/87986955/da451bd9-23d2-4915-bc4e-26c6f83bbb05">